CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # CHANGELOG
 
+### 3.10.5
```

Comment:
This is probably more of a minor change than a patch. We are adding new functionality here.

Reply:
I.e. `3.11.1`

Reply:
Done!